src/window.rs

```diff
@@ -1379,3 +1376,19 @@ impl Default for UserAttentionType {
         UserAttentionType::Informational
     }
 }
+
+/// ## Platform-specific
+///
+/// - **iOS / Android / Web / Wayland:** Unsupported.
+#[derive(Debug, PartialEq, Eq, Clone, Copy)]
+pub enum WindowLevel {
+    /// The window will always be below other windows,
+    ///
+    /// ## Platform-specific
+    ///
+    /// - **Windows**: There is no guarantee that the window will be the bottom most but it will try to be
```

Comment:
In what scenario does it cause issues?

Reply:
I was thinking maybe when two windows try to be the bottom most but this could be the same for any OS and also for `AlwaysOnTop`